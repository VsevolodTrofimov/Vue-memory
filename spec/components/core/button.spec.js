import Vue from 'vue'
const VSSR = require('vue-server-renderer').createRenderer()

import coreButton from '@/src/components/core/Button/Button.vue'


Vue.config.productionTip = false


describe('Components', () => {
  describe('Core Button', () => {
    it('matches snapshot', () => {
      const vm = new Vue({
        render: h => h(coreButton)
      })

      VSSR.renderToString(vm, (err, str) => {
        expect(str).toMatchSnapshot()
      })
    })
  })
})