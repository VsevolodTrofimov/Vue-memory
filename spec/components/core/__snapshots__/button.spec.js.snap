// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Components Core Button matches snapshot 1`] = `<button class="btn"></button>`;
